plugins/modules/aci_l3out_logical_node.py

```diff
@@ -333,7 +372,12 @@ def main():
                     previous_loopback_address = child.get("l3extLoopBackIfP", {}).get("attributes", {}).get("addr")
                     child_configs.append(dict(l3extLoopBackIfP=dict(attributes=dict(addr=previous_loopback_address, status="deleted"))))
             elif loopback_address:
-                child_configs.append(dict(l3extLoopBackIfP=dict(attributes=dict(addr=loopback_address))))
+                loopback_address_config = dict(l3extLoopBackIfP=dict(attributes=dict(addr=loopback_address), children=[]))
+                if mpls_transport_loopback_address:
+                    loopback_address_config["l3extLoopBackIfP"]["children"].append(
+                        dict(mplsNodeSidP=dict(attributes=dict(loopbackAddr=mpls_transport_loopback_address, sidoffset=sid)))
+                    )
+                child_configs.append(loopback_address_config)
```

Comment:
maybe a stupid question but is It possible to add multiple SIDs on the APIC with one loopback address (L3extLoopBackIfP)? If so, why not doing a separate module for `mplsNodeSidP`?

Reply:
No not possible.